frontend/src/components/Release.tsx

```diff
@@ -0,0 +1,85 @@
+import { faCalendar, faFolderOpen } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Tooltip } from '@heroui/tooltip'
+import Image from 'next/image'
+import Link from 'next/link'
+import { useRouter } from 'next/navigation'
+import React from 'react'
+import type { Release as ReleaseTypes } from 'types/release'
+import { formatDate } from 'utils/dateFormatter'
+import { TruncatedText } from 'components/TruncatedText'
+
+interface ReleaseProps {
+  release: ReleaseTypes
+  showAvatar?: boolean
+}
+
+const Release: React.FC<ReleaseProps> = ({ release, showAvatar = true }) => {
+  const router = useRouter()
+  const { name, tagName, publishedAt, author, organizationName, repositoryName } = release
+
+  return (
+    <div className="mb-4 w-full rounded-lg bg-gray-200 p-4 dark:bg-gray-700">
+      <div className="flex w-full flex-col justify-between">
+        <div className="flex w-full items-center">
+          {showAvatar && author && (
+            <Tooltip
+              closeDelay={100}
+              content={author.name || author.login}
+              id={`avatar-tooltip-${tagName}`}
+              delay={100}
+              placement="bottom"
+              showArrow
+            >
+              <Link
+                className="flex-shrink-0 text-blue-400 hover:underline"
+                href={author.login ? `/members/${author.login}` : '#'}
+              >
+                <Image
+                  alt={author.name || author.login}
+                  className="mr-2 h-6 w-6 rounded-full"
+                  height={24}
+                  src={author.avatarUrl}
+                  width={24}
+                />
+              </Link>
+            </Tooltip>
+          )}
+          <h3 className="flex-1 overflow-hidden text-ellipsis whitespace-nowrap font-semibold">
+            <Link
+              className="text-blue-400 hover:underline"
+              href={`https://github.com/${organizationName}/${repositoryName}/releases/tag/${tagName}`}
```

Comment:
Also, the link is currently broken:
<img width="379" height="35" alt="Screenshot 2025-08-16 at 8 19 38 PM" src="https://github.com/user-attachments/assets/3dc20fb6-91c1-4d08-818a-622c6708a3f7" />